openmdao/docs/openmdao_book/features/building_blocks/components/explicit_func_comp.ipynb

```diff
@@ -26,10 +26,9 @@
     "\n",
```

Comment:
<p>I think it would be worthwhile to include a small example of <code>use_jit</code>  here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
added